development/.env

```diff
@@ -4,3 +4,4 @@ DB_NAME=wordpress
 
 #here you can switch between mysql/mariadb
 DATABASE=mariadb
+HOSTNAME=firsthost.test
```

Comment:
Perhaps we can call this `MAILHOG_HOSTNAME` so the purpose is clear:

```suggestion
MAILHOG_HOSTNAME=firsthost.test

```

Reply:
@brylie  Yes indeed that would be better